src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -0,0 +1,578 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Preload video and audio for faster playback.
+
+{# wf_published_on: 2017-07-18 #}
+{# wf_updated_on: 2017-07-18 #}
+
+# Fast Playback with Video Preload {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Faster playback means more people watching your video. That's a known fact.
```

Comment:
nit: Perhaps "faster playback start"? (To prevent possible confusion of faster playback with playbackRate)